coursedata/level-defaults/bg.yaml

```diff
@@ -34,55 +37,629 @@
+         print '5 минус 5 е равно на ' 5 - 5
+         print '5 пъти по 5 е равно на ' 5 * 5
+         ```
+    commands:
+    -   name: "print"
+        explanation: "Принтирай точно думите в кавички"
+        example: "Пример: print '5 пъти по 5 е равно на ' 5 * 5"
+        demo_code: "print '5 пъти по 5 е равно на ' 5 * 5"
+    -   name: "ask"
+        explanation: "Програмирай задача с калкулация и въвеждане на отговор."
+        example: "Пример: решение is ask 'Колко прави 10 + 10?'"
+        demo_code: |-
+            решение is ask 'Колко прави 10 + 10?'
+            if решение is 20 print 'Да!' else print 'Невярно'
+    -   name: "`ask` and `if` turtle"
+        explanation: "Запитай въвеждане на брой ъгли"
+        example: "Пример: ъгли is ask 'Колко ъгли? (част първа)'"
+        demo_code: |-
+            ъгли is ask 'Колко ъгли?'
+            angle is 360 / ъгли
+            forward 50
+6:
+    start_code: "repeat 3 times print 'Да пишеш код на Хеди е забавно!'"
+    intro_text: |
+        Ниво 6! Командите `ask`, `print` и `if`...`else` засега остават непроменени.
+        Новата команда за теб е `repeat`. Repeat на английски значи "повтарям". С нея
+        можеш да инструктираш компютъра да изпълни твоя код много пъти.
+
+        ## Примерен код на Хеди
+        ```
+        repeat 3 times print 'Да пишеш код на Хеди е забавно!'
+        ```
+    commands:
+    -   name: "print"
+        explanation: "Принтирай точно думите в кавички"
+        example: "Пример: print 'Hello welcome to Hedy.'"
+        demo_code: "print 'Hello welcome to Hedy.'"
+    -   name: "ask"
+        explanation: "Запитай с `ask` за любимия спорт."
+        example: "Пример: спорт is ask 'Кой е любимият ти спорт?'"
+        demo_code: |-
+            спорт is ask 'Кой е любимият ти спорт?'
+            print 'Значи обичаш ' спорт '.'
+    -   name: "if"
+        explanation: "Запитай пак за любимия спорт и използвай `if`, за да дадеш специален резултат ако въведената дума е футбол."
+        example: "Пример: спорт is ask 'Кой е любимият ти спорт?'"
+        demo_code: |-
+            спорт is ask 'Кой е любимият ти спорт?'
+            if спорт is футбол repeat 3 times print 'Футбол, футбол и пак футбол!'
+            else print 'Значи обичаш ' спорт '.'
+    -   name: "repeat - turtle"
+        explanation: "Програмирай цикъл с командата `repeat`."
+        example: "Пример: repeat 3 times forward 10"
+        demo_code: |-
+            repeat 3 times forward 20
+7:
+    start_code: |-
+        repeat 5 times
+            print 'Здравейте младежи!'
+            print 'Това ще се принтира 5 пъти.'
+    intro_text: |
+        Командите `ask` и `print` вече познаваш добре, но `if` и `repeat` се променят!
+        You can now execute groups of code together, but you will have to *indent* the code.
+        That means putting four spaces at the beginning of the line. You will also have to indent when you just want to create a block of one line.
```

Comment:
Simply removing everything below this should fix the tests

Reply:
Right, thanks! Following your tip, and a new pull request (in the mean time, on my local machine, I need to find out why the python tests command is failing with..."No module named pytest")

Reply:
It's actually the same Pull Request, but with the new change, commit 8ade052.

Reply:
That is because a lot changed in the mean time incl our test runner. I suspect an update from mail will fix that too.

Reply:
Good news Felienne! Tests are passing :-). Will please merge this bit? :-D

Reply:
Yes, will do!